xbmc/profiles/ProfileManager.cpp

```diff
@@ -513,7 +513,8 @@ bool CProfileManager::DeleteProfile(unsigned int index)
   item->m_bIsFolder = true;
   item->Select(true);
 
-  CGUIComponent *gui = CServiceBroker::GetGUI();
+  CGUIComponent* gui = CServiceBroker::GetGUI();
+
```

Comment:
Revert this changes, please. This is only leftover from a previous iteration of your PR. You do not want to change profile manager.